bot/exts/utilities/color.py

```diff
@@ -0,0 +1,229 @@
+import colorsys
+import json
+import pathlib
+import random
+from io import BytesIO
+
+from PIL import Image, ImageColor
+from discord import Embed, File
+from discord.ext import commands
+from discord.ext.commands.errors import BadArgument
```

Comment:
IMO, just reference this as commands.BadArgument

There are way too many from imports here for my liking